### ISSUE TITLE
default features not marked in the annotation interface

### ISSUE THREAD

**Issue by kabikaj** [kcl-ddh/digipal#52]
Hi again! I selected several default features for components in the admin site, but I don't see them selected in the box that pop-ups with all components and features when I annotate an allograph in an image.

**Comment by pastokes** [comment]
Correct: the default features were never fully implemented. It would be easy enough to do but unfortunately we no longer have any resources to work on this. Contributions to the code are extremely welcome!
